src/main/java/org/jenkinsci/backend/confluence/pageremover/PageNotification.java

```diff
@@ -17,22 +19,26 @@
     /**
      * "added", "edited", etc.
      */
-    public final String action;
+    public String action;
```

Comment:
Not like this is all that matters here, but just curious why you made it non-final.